tools/clang/test/DXC/Passes/SimplifyCFG/simplifycfg-uaf-select-condition.hlsl

```diff
@@ -0,0 +1,59 @@
+// RUN: %dxc -T cs_6_6 %s | FileCheck %s
+
+// This was resulting in an ASAN use-after-free in SimplifyCFG
+// during SimplifyTerminatorOnSelect because it was deleting
+// a PHI node with an input value that the pass later emits
+// (the select condition value).
+// We simply make sure this compiles here, but see
+// simplifycfg-uaf-select-condition.ll for the IR-level test.
```

Comment:
Instead of adding a test that doesn't really verify correctness of the output, can we put this source in a comment in the IR test?

Reply:
Done.